src/current/v23.2/row-level-ttl.md

```diff
@@ -26,7 +26,9 @@ At a high level, Row-Level TTL works by:
 - Issuing batched [`DELETE`]({% link {{ page.version.version }}/delete.md %}) statements for the expired rows.
 - As part of the above process, deciding how many rows to [`SELECT`]({% link {{ page.version.version }}/select-clause.md %}) and [`DELETE`]({% link {{ page.version.version }}/delete.md %}) at once in each of the above queries.
 - Running the SQL queries described above in parallel as [background jobs]({% link {{ page.version.version }}/show-jobs.md %}).
-- To minimize the performance impact on foreground application queries, the background deletion queries are rate limited; they are also submitted at a lower priority level using the [admission control system]({% link {{ page.version.version }}/admission-control.md %}).
+- To minimize the performance impact on foreground application queries, the background deletion queries are rate limited; they are also submitted at a lower priority level using the [admission control system]({% link {{ page.version.version }}/admission-control.md %}). When foreground traffic increases, CockroachDB will reduce the resources allocated to TTL deletes to handle the foreground traffic. When foreground traffic decreases, CockroachDB will increase the resources allocated to TTL deletes.
+
+- {% include_cached new-in.html version="v23.2" %} Latency of row-level TTL queries is further reduced by using the elastic CPU limiter, which dynamically controls the total CPU percentage used by row-level TTL reads. The elastic CPU limiter can be disabled for row-level TTL queries by setting the `kvadmission.ttl_read_elastic_control.enabled` and `sqladmission.low_pri_read_response_elastic_control.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) to `false`.
```

Comment:
is `kvadmission.ttl_read_elastic_control.enabled`` a typo? i don't see this defined anywhere in the code and i am not familiar with it.

Reply:
i think this should be `kvadmission.low_pri_read_elastic_control.enabled`

Reply:
It's from this PR, linked to from a previous comment from Dikshant. https://github.com/cockroachdb/cockroach/pull/108815

Reply:
ah i see, that PR was still in review at that time. an earlier draft called it `kvadmission.ttl_read_elastic_control.enabled` but it is now `kvadmission.low_pri_read_elastic_control.enabled` and was merged with that name.